extensions/grpc/src/main/java/com/hazelcast/jet/grpc/impl/GrpcUtil.java

```diff
@@ -52,8 +52,8 @@ public static Throwable translateGrpcException(Throwable exception) {
      * Tries orderly shutdown first {@link ManagedChannel#shutdown()}, then forceful shutdown
      * {@link ManagedChannel#shutdownNow()}.
      */
-    public static void shutdownChannel(ManagedChannel channel, ILogger logger) throws InterruptedException {
-        if (!channel.shutdown().awaitTermination(1, SECONDS)) {
+    public static void shutdownChannel(ManagedChannel channel, ILogger logger, long timeout) throws InterruptedException {
```

Comment:
Javadoc has a grammar error: "Shutdowns". Should be `Shuts down {@link ManagedChannel}.`

Reply:
Most of the internet agrees with you, I bet both will be correct in 20 years! :-)